posthog/migrations/0194_set_property_type_for_time.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.5 on 2021-12-20 15:38
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0193_auto_20211220_1530"),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            sql="""
+    update posthog_propertydefinition
+    set property_type = 'DateTime', property_type_format='unix_timestamp'
+    where name = '$time'
+        """,
+            reverse_sql="""
+                update posthog_propertydefinition
+    set property_type = null, property_type_format=null
+    where name = '$time'
+                """,
+        )
+    ]
```

Comment:
This migration will now run only once... but what about for users who install a fresh posthog instance, run all the migrations as needed, and only then ingest their first event which contains this property?

Reply:
Yep, going to add smarts later on to do detection and allow editing